apmpackage/apm/data_stream/sampled_traces/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -0,0 +1,7 @@
+---
+description: Pipeline for ingesting APM tail-sampled trace IDs.
+processors:
+  - rename:
+      field: observer.id
+      target_field: agent.ephemeral_id
```

Comment:
ECS says:
>This id normally changes across restarts, but `agent.id` does not.

I believe semantically `observer.id` -> `agent.id` would be more correct.

Reply:
The ID changes on restart now - it's generated at process init.